test/extended/storage/csi-secret-store.go

```diff
@@ -0,0 +1,131 @@
+package storage
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/google/uuid"
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8simage "k8s.io/kubernetes/test/utils/image"
+	admissionapi "k8s.io/pod-security-admission/api"
+
+	"github.com/openshift/library-go/pkg/build/naming"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+const (
+	csiSecretsStoreDriver = "secrets-store.csi.k8s.io"
+	testContainerName     = "test"
+	podWaitTimeout        = 2 * time.Minute
+)
+
+var _ = g.Describe("[sig-storage][Feature:SecretsStore][Serial]", func() {
+	defer g.GinkgoRecover()
+	var (
+		oc                     = exutil.NewCLIWithPodSecurityLevel("secrets-store-test-ns", admissionapi.LevelPrivileged)
+		ctx                    = context.Background()
+		baseDir                = exutil.FixturePath("testdata", "storage", "csi-secret-store")
+		e2eSecretProviderClass = filepath.Join(baseDir, "e2e-provider-secretproviderclass.yaml")
+	)
+
+	g.BeforeEach(func() {
+		exutil.PreTestDump()
```

Comment:
nit: `PreTestDump` doesn't seem to do anything.

Reply:
Heh, whoops, I removed it from the storage tests.